sample/src/extension.ts

```diff
@@ -375,7 +375,7 @@ ${JSON.stringify(result) ?? 'undefined'}`);
         try
         {
             const result: IDotnetListVersionsResult | undefined = await vscode.commands.executeCommand('dotnet.recommendedVersion', { listRuntimes: getRuntimes });
-            vscode.window.showInformationMessage(`Recommended SDK Version to Install: ${result?.at(0)?.version}`);
+            vscode.window.showInformationMessage(`Recommended SDK Version to Install: ${result?.[0]?.version}`);
```

Comment:
Also, I'm curious to know why the recommended version would by null. Can that actually happen and if so, should that be an error or use a different message?

Reply:
Good question - that is if there are no dotnet packages on the distro feed, and or it could not connect to the internet to find any version to recommend to the user. In the rare case their internet comes back after the UI popup, then they can still type in a version and it will work